docs/guide/docs/tutorials.md

```diff
@@ -109,7 +109,7 @@ To enable Postgres, you can edit the configuration or pass 2 environment variabl
 Botpress allows you to build a powerful tool for autonomous communication with your users.
 However there may be cases where it is difficult or very resource-consuming to implement a conversation flow within the bot. At this point you may consider having a human take over the conversation and continue to communicate with your user.
 
-The [Human-in-the-Loop (hitl)](https://github.com/botpress/botpress/tree/master/modules/hitl) module allows you to do just that!
+The [Human-in-the-Loop (hitl)](https://github.com/botpress/botpress/tree/master/modules/.hitl) module allows you to do just that!
```

Comment:
this one is still not supported for 11.x though ...

Reply:
I agree, but I'm just fixing links within this PR.